plugins/php/php_fpm_status

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# 	Plugin to monitor php-fpm process manager, php-fpm 5.3.3 is required
```

Comment:
Please reformat the documentation into the perlpod format (look for `=head1` in other plugins).

And please clarify the version requirement: is it *exactly* v5.3.3 or are later versions supported, too?